pom.xml

```diff
@@ -80,6 +92,16 @@
       <artifactId>grpc-context</artifactId>
       <version>${grpc.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
```

Comment:
Should this really be brought in from the Spring Boot BOM? I am concerned about mixing core Spring modules from different release versions